src/scenes/sceneChanger.tsx

```diff
@@ -0,0 +1,35 @@
+import React, { ReactNode } from "react";
+import styles from "./sceneChanger.module.scss";
+import type { currentScene } from "../store/appControl/appControlSlice";
+import { useSelector } from "react-redux";
+import { currentSceneSelector } from "../store/appControl/selectors";
+
+interface SceneChangerProps {
+  children: ReactNode;
+}
+
+export function SceneChanger({ children }: SceneChangerProps) {
+  const scene = useSelector(currentSceneSelector);
+
+  const arrayOfChildren = React.Children.toArray(children);
```

Comment:
why do you do it like this? Quite uncommon approach

Reply:
I don't know how to choose only one child to display from all passed to the component. This:  `return children[0]` doesn't work.